nodezator/fontsman/cache.py

```diff
@@ -135,6 +146,8 @@ def get_font(font_path, desired_height):
     ### for the font size (here we use the desired height)
     ### and the actual height of a produced surface
 
+    raise_if_unreachable(desired_height, desired_height)
```

Comment:
note: `pygame.font.Font` appears to fail when requesting font sizes of 2^16 or higher. i presume font heights are limited to 16 bits.